### ISSUE TITLE
Cannot build the project

### ISSUE THREAD

**Issue by Sirozha1337** [syl22-00/pocketsphinx.js#119]
Hi, 
I'm trying to build the project. Cmake's output looks like the build was successful, but "pocketsphinx.js" file doesn't appear in the build folder. It has all the .vcxproj files and three folders "CMakeFiles", "include" and "rm1_200".

OS: Windows 10
Visual Studio 2017 Community Edition 
Cmake: 3.11.0-rc1
Emcsripten: 1.37.34
Clang version: 5.0.0

Cmake output:
```
The C compiler identification is MSVC 19.12.25835.0
The CXX compiler identification is MSVC 19.12.25835.0
Check for working C compiler: D:/Soft/VisualStudio/VC/Tools/MSVC/14.12.25827/bin/Hostx86/x86/cl.exe
Check for working C compiler: D:/Soft/VisualStudio/VC/Tools/MSVC/14.12.25827/bin/Hostx86/x86/cl.exe -- works
Detecting C compiler ABI info
Detecting C compiler ABI info - done
Check for working CXX compiler: D:/Soft/VisualStudio/VC/Tools/MSVC/14.12.25827/bin/Hostx86/x86/cl.exe
Check for working CXX compiler: D:/Soft/VisualStudio/VC/Tools/MSVC/14.12.25827/bin/Hostx86/x86/cl.exe -- works
Detecting CXX compiler ABI info
Detecting CXX compiler ABI info - done
Detecting CXX compile features
Detecting CXX compile features - done
Copying D:/Projects/pocketsphinx.js/am/rm1_200 to binary dir
Configuring done
Generating done
```

CMakeOutput.log:
```
The target system is: Emscripten - 1 - x86
The host system is: Windows - 10.0.16299 - AMD64
```

**Comment by syl22-00** [comment]
CMake itself does not build, it creates the build for whatever build tools you have on your system. In your case it seems like it created a Visual Studio solution, so you should probably launch a build there.

**Comment by Sirozha1337** [comment]
Thanks for the quick answer!

I opened generated solution file in Visual Studio, it didn't have the right include directories in project file and when I added Emscripten "include" directory, I got this error: 'Including <emscripten/val.h> requires building with -std=c++11 or newer!', though it is set to "-std=c++latest", changing this option doesn't have any effect at all. I replaced "__cplusplus < 201103L" with "1 < 1" to let my code compile and got a bunch of errors(119) in the val.h file.

Is this is even possible to compile in the latest Visual Studio? 
Is there some VM or docker image I can use for compilation of this library?
